Makefile

```diff
@@ -1,4 +1,10 @@
-.PHONY: all doc doc-man doc-latex docclean
+.PHONY: all allpf doc doc-man doc-latex docclean pfs
+
+# List all platforms
+pfs := $(foreach dir,$(wildcard platforms/*/Makefile),$(subst Makefile, , $(dir)))
+
+all: pfs
```

Comment:
all: $(pfs) ?

Reply:
Yes i'm not even sure that this dependency is needed as existing platforms are listed into the variables...

Reply:
Agree. Looking at the macro expansion, this target will generate multiple sub-make instances. Agree, no need to add target dependencies, this can be handled in sub-make instances.